.github/ISSUE_TEMPLATE/usability.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Usability Improvement
+about: Make Bevy more fun to use!
+title: ''
+labels: C-Usability, S-Needs-Triage
+assignees: ''
+---
+
+## What inconsistency, frustration or unpleasant surprise did you encounter?
+
+Try to be as clear as you can.
```

Comment:
I don't think this advice is particularly helpful. I think it is safe to assume that users know that it is their job to describe their problem.